### PR TITLE
1x engineers need to check their code multiple times...

### PR DESCRIPTION
**Pull request by ahmadnaufal**

### CODE REVIEW

File: docs/index.html [hunk 1]
```diff
@@ -42,6 +42,7 @@ <h2 class="title">A 1x Engineer...</h2>
         <li>Appreciates code reviews from their peers that are constructive and helpful.</li>
         <li>Is open to learning.</li>
         <li>Asks for help when they need it.</li>
+        <li>Double-checking (or quadruple-checking?) their running code in their local machines.</li>
```
Comment (daneah) [inline docs/index.html]: For verb consistency! 😄 

```suggestion
        <li>Double-checks (or quadruple-checks?) their running code in their local machines.</li>
```

Comment (ahmadnaufal) [reply docs/index.html]: Ah, thanks mate!

**Comment by bnb** [comment]
What is the underlying intent here? Double / quadruple checks for what purpose?

**Comment by cherscarlett** [comment]
Hey @ahmadnaufal 👋thank you for your contribution here! Can we clarify the line a bit here? I _think_ what you are going for is that it's okay to second-guess yourself, but the line doesn't really read that way.

**Comment by bnb** [comment]
Going to close since we've not heard back from the original PR author. @ahmadnaufal if you find the time to update, please feel free to commit and comment and we'll re-open ❤️

**Pull request closed without merging**